Support/Scripts/install-android-emulator.sh

```diff
@@ -32,7 +32,7 @@ package_list=()
 
 # This is required due to dependency issues in the sdk. Hopefully this can be removed in future
 # versions of the sdk.
-if [ "${api_level}" -gt 23 ]; then
```

Comment:
I'm not sure you should remove these quotes. If the argument is actually a numeric argument, it won't make a difference, but if the argument is not (error case), the quotes will prevent `[` from failing with stupid errors.